ehrenamt-justiz-eai/README.md

```
@@ -46,18 +46,30 @@
 
 [ehrenamt-justiz-soapui-project.xml](https://github.com/it-at-m/ehrenamt-justiz/blob/main/ehrenamt-justiz-eai/src/test/resources/ehrenamt-justiz-soapui-project.xml)
 
+Example of an EAI call in docker desktop with the parameters first name, last name and date of birth:
+
+<p>
+    <img src="../docs/images/SoapUI_ewosuche_docker.PNG"  alt="Online application"/>
+</p>
+
+Example of an EAI call in docker desktop with the parameter om:
+
+<p>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: p

(MD033, no-inline-html)